toxav/video.c

```diff
@@ -34,6 +34,8 @@
 #include "../toxcore/network.h"
 
 #define MAX_DECODE_TIME_US 0 /* Good quality encode. */
+#define MAX_ENCODE_TIME_US ((1000 / 24) * 1000)
```

Comment:
I don't merge anything that doesn't have @irungentoo's sign off. I also
haven't had time to test it so I haven't been bugging him to merge in
either.

@wiiaam @mannol @subliun any of you had a chance to test this yet?

On Jul 22, 2016 03:51, "vassad" <notifications@github.com> wrote:

> In toxav/video.c
> <https://github.com/irungentoo/toxcore/pull/1606#discussion_r71861045>:
>
> > @@ -34,6 +34,8 @@
> >  #include "../toxcore/network.h"
> >
> >  #define MAX_DECODE_TIME_US 0 /* Good quality encode. */
> > +#define MAX_ENCODE_TIME_US ((1000 / 24) * 1000)
>
> So... Would you mind merging it? :) Or you are not the main merger?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/irungentoo/toxcore/pull/1606/files/185318bbce339f1ee908a46c3f4acfdae93a9198#r71861045>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAO20JyofDW4RUAQ5_pnfHcuoH1WZYZOks5qYKCmgaJpZM4JPAmO>
> .
>